src/environments/environment.prod.ts

```diff
@@ -220,6 +220,13 @@ export const CONSTANTS: Constants = {
       address: ["KT1DuZFNv6JwwA7kS8RXvoQiANm3L4dfn6qP"],
       thumbnailUrl: "../../../assets/img/alias/0xC0FACE-thumb.jpg",
       link: '', shouldDisplayLink: DisplayLinkOption.None
+    },
+    'ChopSumo': {
+      name: "Chop Sumo",
+      address: ["KT1Wm4Cegd7wW6MYxEhg6AH5iFX48y65nWvv"],
+      thumbnailUrl: "../../../assets/img/alias/chopsumo.png",
+      link: "https://chopsumo.xyz", shouldDisplayLink: DisplayLinkOption.None,
+      category: ["collectibles"]
     }
```

Comment:
address: ["KT1DuZFNv6JwwA7kS8RXvoQiANm3L4dfn6qP"],
      thumbnailUrl: "../../../assets/img/alias/0xC0FACE-thumb.jpg",
      link: '', shouldDisplayLink: DisplayLinkOption.None
    },
    'CyberKidz Club': {
      name: "CyberKidz Club",
      address: ["tz1Z5yLks3yX4vk8QR7AsCto3GmgaJnx7dRA"],
      thumbnailUrl: "../../../assets/img/alias/cyberkidzclub.png",
      link: "https://cyberkidzclub.xyz", shouldDisplayLink: DisplayLinkOption.None,
      category: ["collectibles"]
      description: "In 2420, only 6666 kids survived the destruction of the global ecosystem caused by the excessive minting of dirty NFTs on proof of work blockchains. CyberKidz Club."